runtime/src/lib.rs

```diff
@@ -772,24 +746,29 @@ pub type XcmOriginToTransactDispatchOrigin = (
 	ParentAsSuperuser<Origin>,
 	// Native signed account converter; this just converts an `AccountId32` origin into a normal
 	// `Origin::Signed` origin of the same 32-byte value.
-	SignedAccountId32AsNative<RococoNetwork, Origin>,
+	SignedAccountId32AsNative<RelayNetwork, Origin>,
+    // Xcm origins can be represented natively under the Xcm pallet's Xcm origin.
+    XcmPassthrough<Origin>,
 );
 
 parameter_types! {
-	pub UnitWeightCost: Weight = 1_000;
+	// One XCM operation is 1_000_000 weight - almost certainly a conservative estimate.
+	pub UnitWeightCost: Weight = 1_000_000;
+	// One UNIT buys 1 second of weight.
+	pub const WeightPrice: (MultiLocation, u128) = (X1(Parent), UNIT);
```

Comment:
why not same as https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/runtime/src/lib.rs#L360

Reply:
@ltfschoen You are pointing to latest master branch.  I followed this
https://github.com/substrate-developer-hub/substrate-parachain-template/blob/polkadot-v0.9.9/runtime/src/lib.rs#L368